hoomd/simulation.py

```diff
@@ -240,8 +240,10 @@ def operations(self, operations):
                 self._operations._unschedule()
                 self._operations._simulation = None
                 operations._simulation = self
-                operations._schedule()
                 self._operations = operations
+                # This must currently be done after setting _operations so that
+                # self.state.update_group_dof runs without erroring
```

Comment:
Suggestion on the comment text. But more importantly, I don't know if this gives a developer enough info to understand why an error would occur. I haven't looked at `update_group_dof` but if it's not obvious, that would be good to describe.
```suggestion
                # The _schedule method must be called after setting _operations so that
                # self.state.update_group_dof runs without errors.
```